cli/start/setup.go

```diff
@@ -11,8 +12,119 @@ import (
 	"github.com/criblio/scope/internal"
 	"github.com/criblio/scope/run"
 	"github.com/criblio/scope/util"
+	"github.com/rs/zerolog/log"
 )
 
+// extract extracts ldscope and scope to scope version directory
+func extract(scopeDirVersion string) error {
+	// Extract ldscope
+	perms := os.FileMode(0755)
+	b, err := run.Asset("build/ldscope")
+	if err != nil {
+		log.Error().
+			Err(err).
+			Msg("Error retrieving ldscope asset.")
+		return err
+	}
+
+	if err = ioutil.WriteFile(filepath.Join(scopeDirVersion, "ldscope"), b, perms); err != nil {
```

Comment:
```suggestion
	if err = os.WriteFile(filepath.Join(scopeDirVersion, "ldscope"), b, perms); err != nil {
```
I am aware that this code is moved but can we switch to os module since "Deprecated: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code" Ref: https://pkg.go.dev/io/ioutil

Reply:
done